Dockerfile.template

```diff
@@ -45,6 +46,10 @@ RUN set -eux; \
     \
     cd /tmp; \
     \
+    wget https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.buster_amd64.deb; \
```

Comment:
we now support ARM as well, so please make this work for both architectures

Reply:
Here's a code snippet which works for us and could be of use I think:
```dockerfile
ARG TARGETPLATFORM
# ...
RUN case ${TARGETPLATFORM} in \
      "linux/amd64") CURRENT_ARCH=amd64 ;; \
      "linux/arm64") CURRENT_ARCH=aarch64 ;; \
    esac \
    && wget https://some.file.tar.gz \
    && tar -xf some.file.tar.gz \
    && cp -v some.file/${CURRENT_ARCH}/bin/some_app \
```

Reply:
@NiklasBr thank you for the hint,
I'll use slightly different code, asking dpkg about current architecture inside image, which allows to get rid of the dependency on `docker buildx` (pimcore docker test task doesn't use it and adding it dramatically increases tests runtime).

```dockerfile
> RUN DPKG_ARCH="$(dpkg --print-architecture)" \
>     && wget https://some.file.tar.gz \
>     && tar -xf some.file.tar.gz \
>     && cp -v some.file/${DPKG_ARCH}/bin/some_app \
> ```